include/transformations/renaming/TestRenameProcessor.h

```diff
@@ -0,0 +1,21 @@
+#ifndef INCLUDE_TRANSFORMATIONS_RENAMING_TESTRENAMEPROCESSOR_H_
+#define INCLUDE_TRANSFORMATIONS_RENAMING_TESTRENAMEPROCESSOR_H_
+
+#include <string>
+
+#include "BaseRenameProcessor.h"
+
+class TestRenameProcessor : public BaseRenameProcessor {
+    /**
+     * Generate a new name by the following rule:
+     * add prefix "test" into the old name
+     */
+ public:
+    static BaseRenameProcessor * buildFromConfig(const YAML::Node &config);
+    TestRenameProcessor();
```

Comment:
Maybe it is better to make the constructor `explicit`

Reply:
I think no because we should use `explicit` if the constructor has at least one argument